### DISCUSSION TITLE
ギャラリーの情報表示検討

### DISCUSSION THREAD

**Discussion by FltSv** [FltSv/GalleryFound#122] [Ideas / アイデアや提案]
話題の集約化とタスク管理のため、機能の仕様検討はIssueで行いたいと思います。

- ギャラリー
  - テイスト
  - 主な取扱い作品 : 選択（絵、版画、写真、陶器）（色分け）
  - 展示形態？ : 貸し or コマーシャル or 不明？

**Comment by FltSv** [comment]
![image](https://github.com/FltSv/GalleryFound/assets/23271500/7ac8146a-93bd-49ec-8227-0e2dd2b2eecb)

**Comment by FltSv** [comment]
![image](https://github.com/FltSv/GalleryFound/assets/23271500/0c7aca72-1140-47a7-84de-997085958dcd)

**Reply by kurokoutsuki** [reply]
> ![image](https://github.com/user-attachments/assets/2210c513-b372-42e5-88e3-1fd6d674c132)

> コマーシャル・ギャラリーは、画廊主が作品の値段を決めるプライマリー・ギャラリーと、顧客の需要に応えるためオークションなどで作品を調達し取り扱うディーラーが運営するセカンダリー・ギャラリーに分けられる。

https://k-daikoku.net/gallery-in-the-world/
> 日本のアートギャラリーは９割くらいがレンタルギャラリーだと言われています。レンタルギャラリーというのは、名前の通り、作家や展覧会をしたい人が日数によって決められた場所の賃料を支払って作品を展示するギャラリーのことです。日本の多くの貸画廊では１週間（実質６日間）のスケジュールで展覧会が組まれています。レンタルギャラリーの中には、企画展をするところもあり、その場合は賃料は発生しませんし、案内状も作成してくれます。

**Reply by FltSv** [reply]
画像のリンクが死んでるよ～～

**Reply by kurokoutsuki** [reply]
パソコンでは大丈夫だったんだけどな、、、アプリの方見たら？になってたな……（画像引用のなんか間違えてた？？）

**Reply by FltSv** [reply]
こっちはパソコンからもアプリからも、画像のリンク先が見つからないって言ってる　入力フォームに画像D&Dが確実かも

**Reply by kurokoutsuki** [reply]
了解〜帰ったらやってみる（8時間後）

**Reply by kurokoutsuki** [reply]
やったけどくそながURLになっちゃった

**Reply by FltSv** [reply]
```
![](画像URL)
```
で画像表示できるよ

**Reply by FltSv** [reply]
Windowsだったら画像右クリック→画像のコピー、入力欄にCtrl+Vで貼り付けできる

**Reply by kurokoutsuki** [reply]
DEKETA

**Comment by hongeipii** [comment]
優先順位としては
①取り扱い作品のジャンル（絵画、写真、陶器など大雑把で可）
②貸しかコマーシャルギャラリーか（コマーシャル兼貸しのところもあるのでどちらがメインかで大丈夫です）

コレクターはギャラリーのテイストを重視しますので上記2点が区別できると便利です

**Comment by kurokoutsuki** [comment]
> 話題の集約化とタスク管理のため、機能の仕様検討はIssueで行いたいと思います。
> 
> * ギャラリー
>   
>   * テイスト
>   * 主な取扱い作品 : 選択（絵、版画、写真、陶器）（色分け）
>   * 展示形態？ : 貸し or コマーシャル or 不明？

ギャラリーの展示形態はGoogleフォームで募るとかの方が少しは楽かもな～＾

**Reply by kurokoutsuki** [reply]
一応作ったけど見れる？？

https://docs.google.com/forms/d/e/1FAIpQLSedY4HBq2vHwdkgkClrQa_7fPYsn4mhTyV7EmqyZyZoU5MJPQ/viewform?usp=header

**Reply by FltSv** [reply]
見れたよ〜！ギャラリー運営者とかに聞いて回る感じ？
ここで収集した情報をDBに打ち込むのは手打ちになるのがちょっと気にはなるけど…

(文章が気になったのでAIにいい感じにしてもらった)
<details>
以下のように改善してみました。より読みやすく、目的や背景が明確になるよう工夫しています。

---

タイトル：ギャラリー展示形態に関するアンケートのお願い

本文：
この度はアンケートにご協力いただき、誠にありがとうございます。
私たち Gallery Found は、作家やコレクターの皆様が自分に合ったギャラリーを見つけやすくするためのアプリ・ウェブサービスを開発しています。現在、アプリ版のベータテストを公開中ですので、ぜひダウンロードしてお試しいただければ幸いです（無料でご利用いただけます）。

今回のアンケートは、各ギャラリー様の「展示形態」についてお伺いするものです。
Gallery Found（[公式サイトはこちら](https://gallery-found.jp/)）では、アプリ上でギャラリーの展示スタイルや方針を明示することで、作家やコレクターがよりスムーズに、自分に合ったギャラリーを見つけられるようにすることを目的としています。

ご協力のほど、どうぞよろしくお願いいたします。
</details>

**Reply by FltSv** [reply]
Gallery Fo
![Screenshot_20250409-220848](https://github.com/user-attachments/assets/b5f91d2d-6d78-46ad-830b-9413f7c5aec8)

**Reply by FltSv** [reply]
フォーム以前の話にはなるけど、アプリ利用者にとってどういう情報がほしいのか、ってところはもう少し情報収集したほうがいいかも
あとから追加で聞いて回るのも非効率だしギャラリーの人にも申し訳ないかと

**Reply by kurokoutsuki** [reply]
まあ確かに、利用者にアンケで聞いてみるのもあるな～
ギャラリーのサイトとか色々見てみたんだけど詳しく書いてないのもあるし、一応書いてあるとこもあったりする（サイトがデザインチックで見づらいのもあるけど）

形だけ作ってみたって感じだから必要になりそうなら…ってかんじ～

**Reply by kurokoutsuki** [reply]
@FltSv これも返答あれば～＾

**Reply by FltSv** [reply]
返答あればと言われても、、方針があやふやな状態で投げられても、しかも業界外のことなのでこの時点で僕が言えることはあんまり無いけど、、、

**Reply by kurokoutsuki** [reply]
アアなるほど！見てたんならよかた！w

**Comment by FltSv** [comment]
@kurokoutsuki 「ギャラリーの表示」と「地図画面」で分かれてるけど、アプリ内にギャラリー表示画面を作るということ？
色々と説明が不足していて何を伝えたいのか理解ができないので、詳しく説明をお願いします

![image](https://github.com/user-attachments/assets/8ee1a886-9d57-4b59-afed-f89d8ba2958b)

**Reply by kurokoutsuki** [reply]
@FltSv   アプリの地図画面上で、ギャラリーの表示をするってことです！！！すみません‼️

**Reply by FltSv** [reply]
背景がグレーだけど、地図上のどの辺りに表示することを想定してる？（というところまで想定して図として起こしてもらわないと実装に困る）
あと全体的に色が暗めだけど、地図はライトモードしか実装してないので、ライトモードの地図に合うような配色にしないといけない

**Reply by kurokoutsuki** [reply]
場所が違うのがあれだけどこんな感じで想定しています！（色々すみません）

![image](https://github.com/user-attachments/assets/0ff1604b-144d-4735-ac3c-2276b02beae9)

**Reply by FltSv** [reply]
白背景に対して文字が見づらいという同じ過ちをまたしている……

**Comment by kurokoutsuki** [comment]
また同じ過ちを…(´;ω;｀)

二つパターン用意したんだけど、どっちが見やすいかな…<br>テーマカラーのみ
![image](https://github.com/user-attachments/assets/76482928-8d74-4711-bd85-f7ccea6525a1)<br>ボタンとか重要なとこだけテーマカラー
![image](https://github.com/user-attachments/assets/cdc0f390-2a26-4ede-bdd9-c312907b047a)

**Reply by FltSv** [reply]
なんかリプライツリーから外れてるけどまあいいやw
下側の重要なとこだけ色付けるのほうが断然見やすいなあ…(あと×ボタンも黒に)

**Reply by kurokoutsuki** [reply]
よくみてなかった…(´;ω;｀)
[Wiki](https://github.com/FltSv/GalleryFound/wiki/%E4%BB%95%E6%A7%98%E3%83%BB%E8%A6%81%E6%9C%9B%E7%AE%A1%E7%90%86)ではっつけたのでご確認んよろしくお願いします
![image](https://github.com/user-attachments/assets/ba17a622-4b10-44ae-b629-6927003106c0)

**Reply by FltSv** [reply]
@kurokoutsuki 取扱作品の表示、どうやって表示する？

**Reply by kurokoutsuki** [reply]
@FltSv 　あ！そうだった
うーん、ギャラリーを省略して隣に取り扱い作品の表示を配置とかどうかな…
![image](https://github.com/user-attachments/assets/88af27e6-49fb-4bba-b80f-70e607d1a439)

**Reply by FltSv** [reply]
- 半角カタカナダサすぎる
- 貸出/コマーシャル、平面/立体/オール、で分類が違うものを隣に並べるのは分かりづらい気がする
- ↓どうして分離しているのか
![image](https://github.com/user-attachments/assets/c46382f2-1181-473c-abed-15ae93f258ec)

**Reply by kurokoutsuki** [reply]
> 半角カタカナダサすぎる
　貸出/コマーシャル、平面/立体/オール、で分類が違うものを隣に並べるのは分かりづらい気がする

うおおなるほど…
こういう感じ…？
![image](https://github.com/user-attachments/assets/231ecfa5-df9d-45b5-b5de-aaf67dbdd276)

>どうして分離しているのか

あれれあれれ地図画面の下にしてたんだけどな…修正します…

**Reply by himarori** [reply]
> うおおなるほど…
こういう感じ…？

横並びを縦並びにしたところで結局並んでることに変わりはなく、全然違う内容が並んで出てること自体が、わかりにくくない？って話です
貸出とかは多分そこのギャラリー自体を借りるときの話で、ジャンルはそこに何が展示されてるかってことですよね？
まったく内容としては違うことなので、それを似たアイコンで書くのは不親切だということが言いたいんだと思います

> あれれあれれ地図画面の下にしてたんだけどな…修正します…

地図画面に移動されたのは確認しましたが、項目で画像を挟む必要は無いと思います

**Reply by kurokoutsuki** [reply]
うおおひまちゃんコメンツありがとう…（すなさんごめんなさい）
頭回らんくてほんとに申し訳ない、読みこんでて大体はわかってきたのんで
こういう感じかなと思ったんだけどいかがかしら…↓
![image](https://github.com/user-attachments/assets/ed5222e3-6402-4eda-9c67-dc497c1d1000)


> 地図画面に移動されたのは確認しましたが、項目で画像を挟む必要は無いと思います

一旦画像を消しました！

**Reply by FltSv** [reply]
当初よりはマシだけど…オールジャンルのやつって選択制だよね？ハッシュタグみたいな見た目にすると複数あるものだと勘違いされない？
あとこの位置に持ってくると↓の余白が気になってくるから別の位置がいいと思う（ギャラリー名が小窓のタイトルみたいなものだと思うけど、タイトルの上に情報があるという位置関係もおかしい）
![image](https://github.com/user-attachments/assets/5c0bd402-9013-44da-b03c-d874aeab612b)

> 一旦画像を消しました！

画像が必要ないとは言ってないけど………

**Reply by kurokoutsuki** [reply]
うお～そうだよね…
なんか位置がすごい悩むけど2点用意してきた（咳で頭回らんかも）
![image](https://github.com/user-attachments/assets/a38174b9-0a6f-4599-978b-f3ec66bb89e8)

![image](https://github.com/user-attachments/assets/2b4eb414-e0d6-4eca-b9bc-744d6c34833f)

[Wiki](https://github.com/FltSv/GalleryFound/wiki/%E4%BB%95%E6%A7%98%E3%83%BB%E8%A6%81%E6%9C%9B%E7%AE%A1%E7%90%86)はっつけました～すまぬ～(´;ω;｀)

**Reply by FltSv** [reply]
貸出とかの位置、ギャラリー名が長い場合の配置に困るになってる

![image](https://github.com/user-attachments/assets/2b4374a3-74fc-40d5-a42e-6cdaab1d47ea)

一番右に寄せるとなるとこうなる、けど高さが無駄に増える要因になりがち

![image](https://github.com/user-attachments/assets/b1373295-658c-4f2e-a91e-ab3439323ce0)

**Reply by FltSv** [reply]
スマホで表示したときに考慮すべき最小幅で表示したらこんな感じ↓

![image](https://github.com/user-attachments/assets/2dde8cf1-c5cf-44c3-b01e-afdcf1bfdf9b)


ボタンがスペース取り過ぎてて崩れてしまってるので、最小幅でも問題なく表示できるようなデザインで考える必要がある

**Reply by kurokoutsuki** [reply]
https://github.com/FltSv/GalleryFound/discussions/122#discussioncomment-13027696
2枚目のギャラリーのおしりにくっつくかんじでもいいと思うんだけどな…

最小限は単語で文字を省略するとか…？？
![image](https://github.com/user-attachments/assets/ce8c9bd1-93ea-4752-94b9-b22a6f0092f2)

**Reply by FltSv** [reply]
> https://github.com/FltSv/GalleryFound/discussions/122#discussioncomment-13027696
> 2枚目のギャラリーのおしりにくっつくかんじでもいいと思うんだけどな…

ギャラリーのおしりにくっついてる方は1枚目の認識だけど2枚目？1枚目と2枚目の認識が入れ替わってるように聞こえてる

> 最小限は単語で文字を省略するとか…？？
> [image](https://github.com/user-attachments/assets/ce8c9bd1-93ea-4752-94b9-b22a6f0092f2)
> 

GoogleMapだけの表示だとまあ頑張れば分かるけど、、見る側に頑張らせるデザインは「デザインとして」良くない
ブランドイメージ(こだわり)もあるにはあるけど、それを優先し過ぎるが上に分かりにくくなるのは本末転倒だと思う
それこそ、ボタンのような見た目が本当に必要なのかどうか、とか、根本的なところから考え直す必要があるような気がする

**Reply by kurokoutsuki** [reply]
> ギャラリーのおしりにくっついてる方は1枚目の認識だけど2枚目？1枚目と2枚目の認識が入れ替わってるように聞こえてる

1枚目だった…見間違いでした(^т  ̫ т^)

> ブランドイメージ(こだわり)もあるにはあるけど、それを優先し過ぎるが上に分かりにくくなるのは本末転倒だと思う
それこそ、ボタンのような見た目が本当に必要なのかどうか、とか、根本的なところから考え直す必要があるような気がする

たしかに頑張らせるのも不親切だったし、ボタンの見た目考えとく…

**Reply by kurokoutsuki** [reply]
アイコン付きでセカンダリーの色みたいな感じにしてみたけどいかがかしら…

![image](https://github.com/user-attachments/assets/ded3f8c8-15c4-495a-b5b1-1d1890a3e15e)

**Reply by FltSv** [reply]
> 
> アイコン付きでセカンダリーの色みたいな感じにしてみたけどいかがかしら…
> 
> [image](https://github.com/user-attachments/assets/ded3f8c8-15c4-495a-b5b1-1d1890a3e15e)
> 


狭い横幅でも表示できるようにと言った話覚えてる？？？？なんで横幅増やすことするんですか？？？？？
今までの会話とか見返したりしてる？？？

**Reply by kurokoutsuki** [reply]
覚えてるよ～！(´;ω;｀)

説明不足だったかも、縮小時にアイコンにするとか思ってたけどやっぱり違ったから
chatgptに投げたら↓がでてきたので
![ChatGPT Image 2025年5月5日 13_40_26](https://github.com/user-attachments/assets/86819b75-c8c5-484e-bb9a-2180c32bfafa)

2つパターン用意したんだけど、いかがかしら…（下の▶GoogleMapで開く～とかの色が悩んでしまっていた）
![image](https://github.com/user-attachments/assets/57e41d63-2c1e-4f61-8e94-7fccd04378bb)
![image](https://github.com/user-attachments/assets/8d836442-c9c4-44e9-b921-c8f6c2db435f)

**Reply by himarori** [reply]
> 覚えてるよ～！(´;ω;｀)
> 
> 説明不足だったかも、縮小時にアイコンにするとか思ってたけどやっぱり違ったから
> chatgptに投げたら↓がでてきたので
> ![ChatGPT Image 2025年5月5日 13_40_26](https://github.com/user-attachments/assets/86819b75-c8c5-484e-bb9a-2180c32bfafa)
> 
> 2つパターン用意したんだけど、いかがかしら…（下の▶GoogleMapで開く～とかの色が悩んでしまっていた）
> ![image](https://github.com/user-attachments/assets/57e41d63-2c1e-4f61-8e94-7fccd04378bb)
> ![image](https://github.com/user-attachments/assets/8d836442-c9c4-44e9-b921-c8f6c2db435f)
> 

- 「休廊」の表示は、仕様上どうしても営業時間のところにプルダウンで表示されるので、別での表記はできません(Google Mapの営業時間と同じ感じです)
- アイコンを使う場合は、もう少しシンプルなものにしてください(これもGoogle Mapがわかりやすいです)
- リンクは「青色＋下線」にしないと、ユーザーがリンクと認識しにくいと思います！
- ChatGPTの提案をそのまま活かしすぎているふうに見えます〜（例：貸出の塗りつぶし、 Google Mapで開く際の三角アイコンなど）
- 横幅が広いままだと、スマホなど画面の狭い端末で表示が崩れてしまうので、そういった画面幅でもキレイに見えるよう調整をお願いします！
- 横幅の可変対応はWeb画面では可能ですが、地図画面ではできないです〜
- 「Google Mapで開く」の文言がわかりづらいので、地図のマークに「詳細」などシンプルなテキストの方が直感的に伝わりやすいです
![Screenshot_20250505-142521.png](https://github.com/user-attachments/assets/28102a44-c1df-4bde-ae3a-0f8703d79907)

**Reply by FltSv** [reply]
補足すると、「Googleマップで開く」→「詳細」ボタン、「ここへの経路」→「経路」ボタン、とする方が直感的で分かりやすいのでは？という話になってます

経路はちょうどGoogle Mapの↓のような感じでアイコンを付ける
![Screenshot_20250505-164648](https://github.com/user-attachments/assets/04f13ab2-c152-4e23-bbad-aa07267b7f35)

詳細ボタンもアイコンを付けたいけど、何のアイコンにするかは未定…

**Reply by kurokoutsuki** [reply]
ひまちゃんReコメンツ助かります…(´;ω;｀)
（後ろの背景にあるピン詳細が見えるけど気にしないでください汗）
洗練された感じってことだったんですかね！？！！（合ってる？？）
コメンツとスクショをもとにつくってみました↓
縦長すぎましたかね…


![image](https://github.com/user-attachments/assets/6617533d-ff5e-4c58-9381-c9bc2a74e2fb)

**Reply by kurokoutsuki** [reply]
今見た！

> 詳細ボタンもアイコンを付けたいけど、何のアイコンにするかは未定

未定であれだけど提案として↓のはどうかな
![image](https://github.com/user-attachments/assets/30214e1c-9936-453a-a96f-518a8a0816c6)

これはただの詳細アイコン
![image](https://github.com/user-attachments/assets/f5b6c154-e75e-4bfb-bb6f-c0b780d40cf1)

**Reply by himarori** [reply]
> ひまちゃんReコメンツ助かります…(´;ω;｀)
> （後ろの背景にあるピン詳細が見えるけど気にしないでください汗）
> 洗練された感じってことだったんですかね！？！！（合ってる？？）
> コメンツとスクショをもとにつくってみました↓
> 縦長すぎましたかね…
> 
> 
> [image](https://github.com/user-attachments/assets/6617533d-ff5e-4c58-9381-c9bc2a74e2fb)
> 

ありがとうございます 今までよりいい感じだと思います〜
洗練ではなくてほんとにただの言いなりになってる感じがすごかったかなーと あくまでもヒントをもらって何がいいか？って考えるのがいいと思います！
あと詳細とか経路のアイコン？はボタンにしたほうがわかりやすいかな〜と思いました
この表現方法だとみんなアイコンとか押しちゃうと思うけど反応しないので…！
Google Mapみたいに「アイコン+詳細」をボタンでくくったほうがわかりやすそうです

**Reply by kurokoutsuki** [reply]
> 洗練ではなくてほんとにただの言いなりになってる感じがすごかったかなーと あくまでもヒントをもらって何がいいか？って考えるのがいいと思います！

ありがとうございます(´;ω;｀)AIを活用するってことですね…大変勉強になりました…（）


> あと詳細とか経路のアイコン？はボタンにしたほうがわかりやすいかな〜と思いました
この表現方法だとみんなアイコンとか押しちゃうと思うけど反応しないので…！
Google Mapみたいに「アイコン+詳細」をボタンでくくったほうがわかりやすそうです

こんな感じでいかがでしょうか！

![image](https://github.com/user-attachments/assets/d3cdc65e-ff54-4c62-8f25-19e6948389b5)

**Reply by himarori** [reply]
>ありがとうございます(´;ω;｀)AIを活用するってことですね…大変勉強になりました…（）

そういうことです！結局見た目とかそれっぽいのは出してくれても、本当に良いかは自分で判断しないといけないと思っているので。。

>こんな感じでいかがでしょうか！

経路とかを青にした方が良くない？って話をしたのはあくまでもリンクにする場合なので、今回だと色浮いて見えるかな～～って感じです
ボタンの色は以前もらったこの色で良いかと思います！
![image](https://github.com/user-attachments/assets/a3352008-4337-46e1-87c7-9fec497532b9)

**Reply by kurokoutsuki** [reply]
> ボタンの色は以前もらったこの色で良いかと思います！

こういうことですかね？！
![image](https://github.com/user-attachments/assets/0582fac4-f331-4ddb-a636-6617f8aa9440)

**Reply by himarori** [reply]
>こういうことですかね？！

そういうことです！ありがとうございます！あとは実装してもらってまた何かあったら連絡します～
